docs/framework/network-programming/tls.md

```diff
@@ -15,34 +15,57 @@ helpviewer_keywords:
 ---
 # Transport Layer Security (TLS) best practices with the .NET Framework
 
-The Transport Layer Security (TLS) protocol is an industry standard designed to help protect the privacy of information communicated over the Internet. [TLS 1.2](https://tools.ietf.org/html/rfc5246) is a standard that provides security improvements over previous versions. TLS 1.2 will eventually be replaced by the newest released standard [TLS 1.3](https://tools.ietf.org/html/rfc8446) which is faster and has improved security. This article presents recommendations to secure .NET Framework applications that use the TLS protocol.
+.NET Framework supports the use of the Transport Layer Security (TLS) protocol to secure network communications.
 
-To ensure .NET Framework applications remain secure, the TLS version should **not** be hardcoded. .NET Framework applications should use the TLS version the operating system (OS) supports.
+## What is Transport Layer Security (TLS)?
 
-This document targets developers who are:
+> [!WARNING]
+> TLS 1.0 and 1.1 has been deprecated by [RFC8996](https://datatracker.ietf.org/doc/rfc8996/). This document will cover TLS 1.2 and TLS 1.3.
+
+The Transport Layer Security (TLS) protocol is an industry standard designed to help protect the privacy of information communicated over the Internet. [TLS 1.3](https://tools.ietf.org/html/rfc8446) is a standard that provides security improvements over previous versions. This article presents recommendations to secure .NET Framework applications that use the TLS protocol.
+
+## Who can benefit from this document?
 
 - Directly using the <xref:System.Net> APIs (for example, <xref:System.Net.Http.HttpClient?displayProperty=nameWithType> and <xref:System.Net.Security.SslStream?displayProperty=nameWithType>).
 - Directly using WCF clients and services using the <xref:System.ServiceModel?displayProperty=nameWithType> namespace.
 
-Consider the following recommendations:
+## TLS support in .NET Framework
+
+Since the .NET Framework is dependent on `Schannel` on Windows, which versions can be negotiated and which version will be used depends on the operating system.
+
+Here is an updated example table showing the highest supported TLS version for different combinations of operating system versions and .NET Framework target versions:
+
+| .NET Framework Target Version | Windows 10 | Windows 11 |
+|-------------------------------|------------|------------|
+| 3.5                           | TLS 1.2    | TLS 1.2    |
+| 4.6.2                         | TLS 1.2    | TLS 1.3    |
```

Comment:
I think this is somewhat confusing IMHO since 4.6 does not support the TLs 13 enum. 
I'm wondering if we should be quiet here and explain later. The "use OS default" is sort of hack IMHO and needs to be combined with the right runtime.

I do like the link to the schannel.

Reply:
But this is targeting version not the runtime version. So I believe we should be fine in that manner.